ext/mysqli/tests/mysqli_connect_oo.phpt

```diff
@@ -10,7 +10,7 @@ require_once 'skipifconnectfailure.inc';
 <?php
     require_once 'connect.inc';
 
-    if ($mysqli = new mysqli($host, $user . 'unknown_really', $passwd . 'non_empty', $db, $port, $socket) && !mysqli_connect_errno())
+    if ($mysqli = @new mysqli($host, $user . 'unknown_really', $passwd . 'non_empty', $db, $port, $socket) && !mysqli_connect_errno())
```

Comment:
Can this at least not be in the condition? It's a bit confusing what we are actually checking here.

Reply:
This is basically the same test as in mysqli_connect.phpt above; we try to connect as unknown user, which should not succeed. If it does, an error message is printed.

I don't understand what you are suggesting, though. Can you please clarify?